source/tutorial/ex6.rst

```diff
@@ -0,0 +1,108 @@
+.. _Exercise 6:
+
+Exercise 6 \\- \ Feature Customization - VerifiedBoot
+---------------
+
+.. note::
+  **In this exercise, we'll learn how to enable and verify VerifiedBoot.**
```

Comment:
I think this exercise needs a bit more explanation. Is the goal here to demonstrate that boot stage signature verification will fail if a user modifies a byte in one of the stages?

Reply:
correct, updated